### ISSUE TITLE
FNX-3404 ⁃ [Bug]Search bar-"Clear search" button not displayed for pre-selected URL

### ISSUE THREAD

**Issue by Diana-Rus** [mozilla-mobile/fenix#10282]
## Steps to reproduce
1. Open a tab with URL.
1. Tap on the search bar.
    1. Expected behavior
The url is highlighted and "clear search" button is displayed in the right side of the search bar. 
    1. Actual behavior
"Clear search" button is missing. The user can only clear through virtual keyboard.

**Note:**

* The "clear search" button is displayed only after using the clear button from the virtual keyboard. 

### Device information
* Android device: Google Pixel 3 XL (Android 9), Samsung Galaxy S9 (Android 8)
* Fenix version: Beta 5.0.0-beta.1 4/29,  Firefox Nightly 4/29

Pixel 3 XL - GIF
![20200429-121936](https://user-images.githubusercontent.com/49859863/80580945-07eadc00-8a15-11ea-8384-3b2719349d58.gif)

**Comment by Diana-Rus** [comment]
Duplicate of: 10291
